code/game/machinery/modular_shield.dm

```diff
@@ -514,16 +518,21 @@
 		return
 	icon_state = "node_on_[panel_open ? "open" : "closed"]"
 
-/obj/machinery/modular_shield/module/node/setDir(new_dir)
-	. = ..()
 
-	disconnect_connected_through_us()
-	if(isnull(shield_generator))
-		return
-	LAZYREMOVE(shield_generator.connected_modules, (src))
-	shield_generator.calculate_boost()
-	shield_generator = null
-	update_icon_state()
+/obj/machinery/modular_shield/module/node/wrench_act(mob/living/user, obj/item/tool)
+
+	if(default_change_direction_wrench(user, tool))
```

Comment:
You should return FALSE if the statement is false, to avoid this indentation.

Reply:
doesnt it automatically return false if it doesnt get a true return?

Reply:
I mean, it should be like this:
```
	if(!default_change_direction_wrench(user, tool))
		return FALSE
```
And everything following it as normal, but with less indentation.

Reply:
oh so you want to check for inverse then do everything else, okay.

Reply:
done